sympy/plotting/plot.py

```diff
@@ -237,6 +237,16 @@ def append(self, arg):
         Plot object containing:
         [0]: cartesian line: x**2 for x over (-10.0, 10.0)
         [1]: cartesian line: x for x over (-10.0, 10.0)
+	
+	    .. plot::
+
+	       >>> from sympy import symbols
+           >>> from sympy.plotting import plot
+           >>> x = symbols('x')
+           >>> p1 = plot(x*x, show = False)
+           >>> p2 = plot(x, show = False)
+           >>> p1.append(p2[0])
+           >>> p1.show()	
```

Comment:
You have the same code twice. Please only have 1 example and 1 plot.

Reply:
I had to add the show = False in the code to prevent multiple plots being shown, so if it's ok for "show = False" showing up in the example code, I can make necassary changes to remove the redundant code.

And here is the screenshot:
![ss1](https://user-images.githubusercontent.com/35144226/46560298-c796ab00-c910-11e8-8b48-f1eee15f2be3.png)